TUTORIAL.md

```diff
@@ -159,6 +159,40 @@ If you encounter 'Connection reset by peer' errors when cloning, run, say, `assi
 (Assigner doesn't have plans for any grading features;
 however, if you are interested in automated grading, check out its sister project, [grader](https://github.com/redkyn/grader).)
 
+### Uploading scores to Canvas
+
+Although Assigner does not support grading, it can be used to upload the scores generated by an automated grader to Canvas.
+Currently, this feature is supported for workflows in which an automated grading script runs as part of a Continuous Integration (CI) job and 
+produces an output file (artifact) containing the score.
+
+There are two methods of uploading these scores:
+1. Use `assigner score all` if you'd like to get the scores for all students, passing the `--upload` flag if you'd like to upload them to Canvas.
+1. Use `assigner score checkout` if you'd like to upload scores one at a time, using an interactive search function to select a student.  
+
+The latter option may be useful in circumstances where an instructor wants to verify attendance as part of a programming lab or test.
+
+An example grading workflow for a conventional assignment might look like the following:
+1. `assigner score all assignment-name` while the assignment is open, to monitor student progress.
+1. `assigner lock assignment-name` once the due date is reached
+1. `assigner score all --upload assignment-name -f path/to/grader_file1 path/to/grader_file2` to upload scores and check whether grading-related files were modified 
+by the student
+
+#### Details on checking the integrity of grading files
+A change to a grading-related file is considered legitimate if it is GPG-signed by a maintainer or owner of the repository.
+Note that only commits made after the assignment is opened to students are checked, so this process is only necessary if 
+grading-related files are changed after an assignment is released.
+
+To use GPG to sign a commit, you must complete the following:
+1. Generate a GPG key if you don't have one.
+1. Associate your GPG key with `git` using the `user.signingkey` config option.
+1. Associate your GPG key with the appropriate repository manager (GitHub, GitLab, etc).
+1. Sign all commits made after the assignment has been opened by passing the `-S` flag to `git commit`.
+
+[GitLab's tutorial](https://docs.gitlab.com/ee/user/project/repository/gpg_signed_commits/) explains the entire process in detail.
+
+Any deviations will be reported by default unless the `--nocheck` flag is passed to any of the `assigner score` subcommands.
```

Comment:
`--noverify`; in general it'd be good to look these back over and make sure the command names match